manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/DispatcherAlarm.java

```diff
@@ -135,6 +135,7 @@ public void run() {
                     }
                 } catch (IgnoreException ignored) {
                 } catch (InterruptedException e) {
+                    Thread.currentThread().interrupt();
```

Comment:
this code should be in the previous catch block. right?

Reply:
Code block in InterruptedException

Reply:
got it.